src/components/CalendarMonthGrid.jsx

```diff
@@ -60,6 +64,19 @@ const defaultProps = {
   monthFormat: 'MMMM YYYY', // english locale
 };
 
+function getCalendarMonthGridWidthStyle(orientation, withWeekNumbers) {
```

Comment:
Yes, I also think that this is not the best solution, but at least the width value is not duplicated in css and js this way.